### PR TITLE
MD-77 || CI Configuration for generating pr builds

### PR DESCRIPTION
**Pull request by creativeprashant007**
ref:[jira](https://readytowork.atlassian.net/browse/MD-77)

### CODE REVIEW

**Comment by creativeprashant007** [comment]
Me too

**Comment by creativeprashant007** [comment]
Comment Api Tested Successfully

**Comment by creativeprashant007** [comment]
Comment Api Tested Successfully

**Comment by creativeprashant007** [comment]
Comment Api Tested Successfully

**Comment by creativeprashant007** [comment]
Comment Api Tested Successfully

**Comment by creativeprashant007** [comment]
Comment Api Tested Successfully

**Comment by creativeprashant007** [comment]
Comment Api Tested Successfully

**Comment by creativeprashant007** [comment]
Comment Api Tested Successfully

**Comment by creativeprashant007** [comment]
Comment Api Tested Successfully

**Comment by creativeprashant007** [comment]
Comment Api Tested Successfully

**Comment by creativeprashant007** [comment]
This is test comment

**Comment by creativeprashant007** [comment]
This is test comment

**Comment by creativeprashant007** [comment]
This is test comment

**Comment by creativeprashant007** [comment]
This is test comment

**Comment by creativeprashant007** [comment]
This is test comment

**Comment by creativeprashant007** [comment]
build/app/outputs/flutter-apk/app-armeabi-v7a-release.apk

**Comment by creativeprashant007** [comment]
Click [here](https://example.com/my-artifact) to access the artifact.

**Comment by creativeprashant007** [comment]
Click [here](https://circleci.com/api/v1.1/project/github/readytowork-org/flutter-skeleton/41/artifacts/build/app/outputs/flutter-apk/app-armeabi-v7a-release.apk) to access the artifact.

**Comment by creativeprashant007** [comment]
Click [here](https://circleci.com/api/v1.1/project/github/readytowork-org/flutter-skeleton/42/artifacts/build/app/outputs/flutter-apk/app-armeabi-v7a-release.apk) to access the artifact.

**Comment by creativeprashant007** [comment]
Click [here](https:///output/job/7930c78d-6e60-453a-ab24-2dad3d53a901/artifacts/0/build/app/outputs/flutter-apk/app-armeabi-v7a-release.apk) to access the artifact.

**Comment by creativeprashant007** [comment]
Click [here](https://output.circle-artifacts.com/output/job/6200f8b9-059b-4860-b0a6-b7a66010150e/artifacts/0/build/app/outputs/flutter-apk/app-armeabi-v7a-release.apk) to access the artifact.

File: lib/config.dart.example
Comment (sawal-timsina) [inline lib/config.dart.example:1]: @creativeprashant007  revert this please

**Comment by creativeprashant007** [comment]
Click [here](https://output.circle-artifacts.com/output/job/49206fef-9fa4-41b8-9442-0aa1c7655ba4/artifacts/0/build/app/outputs/flutter-apk/app-armeabi-v7a-release.apk) to download the app.

**Comment by creativeprashant007** [comment]
Click [here](https://output.circle-artifacts.com/output/job/b71196df-03a8-4006-aa19-49dc353ae425/artifacts/0/build/app/outputs/flutter-apk/app-armeabi-v7a-release.apk) to download the app.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          name: Add Google services to App
+          command: |
+            echo "$GOOGLE_APPLICATION_CREDENTIALS" | base64 --decode > android/app/google-services.json
+
+      - run:
+          name: Assemble release build
+          command: |
+            TAG_NAME=$(echo $CIRCLE_TAG | cut -d'-' -f1)
+            VERSION_CODE=$(echo $TAG_NAME | cut -d'v' -f2)
+            flutter build apk --split-per-abi --release --build-name=$VERSION_CODE
```
Comment (sawal-timsina) [inline .circleci/config.yml]: @creativeprashant007  let's not split the apks, as it might not run on everyone's device

**Review by sawal-timsina — CHANGES_REQUESTED** [review]
Remove .example from config import
Remove firebase config
Add Branch name in app label

**Comment by creativeprashant007** [comment]
Click [here](https://output.circle-artifacts.com/output/job/0527be6e-bdfb-46bf-805c-f9f13ce8357c/artifacts/0/build/app/outputs/apk/release/app-release.apk) to download the app.

**Comment by creativeprashant007** [comment]
Click [here](https://output.circle-artifacts.com/output/job/fcf310f3-ed6b-4b5e-9fbb-27c5c7cb50ef/artifacts/0/build/app/outputs/apk/release/app-release.apk) to download the app.

**Review by sawal-timsina — APPROVED** [review]
LGTM

**Pull request closed without merging**